jekyll/_cci2/machine-runner-3-configuration-reference.adoc

```diff
@@ -233,6 +233,25 @@ runner:
   mode: continuous
 ```
 
+When using `single-task` mode, the expected behaviour is that the runner process will while the machine instance will continue. However you may find it useful to terminate the machine instance itself.
+You can do this on Linux based machines by adding the following to the systemd unit file:
+
+```yaml
+ExecStopPost=shutdown now -h
+Restart=no
+User=root
+```
+
+However to avoid running as root we recommend stepping down the `circleci` user using `runner.command_prefix` as in our xref:https://circleci.com/docs/machine-runner-3-configuration-reference/#runner-command-prefix[example above].
```

Comment:
```suggestion
However to avoid running as root we recommend stepping down the `circleci` user using `runner.command_prefix` as in our <<runner-command-prefix,example above>>.
```